tools/astropytools/fits2table.xml

```diff
@@ -20,31 +20,31 @@ with fits.open("$input_fits", memmap=True) as hdu_list:
     table = Table(hdu.data)
 
     # write to a CSV file
-    table.write('$output_csv', delimiter='$output_delimiter', format='ascii', overwrite=True)
+    table.write('$output', delimiter='$output_delimiter', format='ascii', overwrite=True)
         </configfile>
     </configfiles>
 
     <inputs>
         <param type="data" name="input_fits" format="fits" label="FITS file to dump"/>
         <param type="integer" name="hdu"  value="1" min="1" label="Select input HDU number"/>
         <param type="select" name="output_delimiter" value="," label="Output delimiter">
-            <option value="," selected="true">"," (coma)</option>
+            <option value="\t" selected="true">"\t" (tabular)</option>
+            <option value=",">"," (coma)</option>
             <option value=" ">" " (space)</option>
-            <option value="\t">"\t" (tab)</option>
         </param>
     </inputs>
     <outputs>
-        <data name="output_csv" format="csv"/>
+        <data name="output" format="csv,ssv,tabular"/>
```

Comment:
outputs can only have ONE format ... you need to use this here to change the format depending on the user selection: https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-outputs-data-change-format-when

Reply:
And I see now ... Galaxy does not support SSV yet, never heard about it until now :)

Reply:
> outputs can only have ONE format ... you need to use this here to change the format depending on the user selection: https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-outputs-data-change-format-when

I was experimenting. Linting did not complain. I guess it just sees it as long type with comas?

Reply:
Mh ... do you know if XSD can check for those things?

Reply:
Maybe, I did not use it, but it seems? https://www.w3schools.com/XML/schema_facets.asp https://stackoverflow.com/questions/8925706/xml-schema-how-to-restrict-attribute-by-enumeration ?

Reply:
It causes warning in the test though but not an error.